geomstats/geometry/hyperbolic.py

```diff
@@ -362,10 +366,12 @@ def random_uniform(self, n_samples=1, bound=1.):
 
         Parameters
         ----------
-        n_samples : int, optional
+        n_samples : int
             Number of samples.
-        bound: float, optional
+            Optional, default: 1.
```

Comment:
1

Reply:
Let's keep the full stop at the end of the sentence?